src/sentry/locale/catalogs.json

```diff
@@ -1,16 +1,16 @@
 {
   "supported_locales": [
-    "bg",
-    "cs",
-    "de",
-    "en",
-    "es",
-    "fi",
-    "fr",
-    "it",
-    "ja",
-    "ru",
-    "zh_CN",
+    "bg", 
```

Comment:
Did you mean to add this space to the end of all the lines?

Reply:
Yes this is autogenerated, see https://bugs.python.org/issue16333